From 4ebcd533bea217a31cd49523793b7992f5696dd9 Mon Sep 17 00:00:00 2001
From: Mikolaj <mikolaj.konarski@gmail.com>
Date: Tue, 5 Jul 2011 18:49:37 +0200
Subject: [PATCH 1/7] add a missing traceSparkCounters invocation

---
 rts/Capability.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rts/Capability.c b/rts/Capability.c
index 3c06f5a..e03a1f2 100644
--- a/rts/Capability.c
+++ b/rts/Capability.c
@@ -621,6 +621,7 @@ yieldCapability (Capability** pCap, Task *task)
         traceEventGcStart(cap);
         gcWorkerThread(cap);
         traceEventGcEnd(cap);
+        traceSparkCounters(cap);
         return;
     }
 
-- 
1.7.4.1

