From 98ff7f6b71ca3de21d2ba13cc40dd6a88c78edd0 Mon Sep 17 00:00:00 2001
From: Mikolaj <mikolaj.konarski@gmail.com>
Date: Fri, 8 Jul 2011 19:43:41 +0200
Subject: [PATCH 4/7] include spark counters in the sparks trace class

---
 rts/Trace.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/rts/Trace.h b/rts/Trace.h
index 285caf6..18542d4 100644
--- a/rts/Trace.h
+++ b/rts/Trace.h
@@ -498,7 +498,7 @@ INLINE_HEADER void traceOSProcessInfo(void)
 INLINE_HEADER void traceSparkCounters(Capability *cap STG_UNUSED)
 {
 #ifdef THREADED_RTS
-    if (RTS_UNLIKELY(TRACE_sched)) {
+    if (RTS_UNLIKELY(TRACE_spark)) {
         traceSparkCounters_(cap, cap->spark_stats, sparkPoolSize(cap->sparks));
     }
 #endif
-- 
1.7.4.1

