From 885d4f616ee9b831b71c3d6601d9f12d6e96336c Mon Sep 17 00:00:00 2001
From: Mikolaj <mikolaj.konarski@gmail.com>
Date: Tue, 5 Jul 2011 18:49:37 +0200
Subject: [PATCH] add a missing traceSparkCounters invocation

---
 rts/Capability.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rts/Capability.c b/rts/Capability.c
index 7982b20..2d11ecc 100644
--- a/rts/Capability.c
+++ b/rts/Capability.c
@@ -621,6 +621,7 @@ yieldCapability (Capability** pCap, Task *task)
         traceEventGcStart(cap);
         gcWorkerThread(cap);
         traceEventGcEnd(cap);
+        traceSparkCounters(cap);
         return;
     }
 
-- 
1.7.4.1

